Resolve per-app docroots at runtime

jacobythwaites opened the pull request:
If `sys.config` references the `priv` dir of apps using absolute paths relative to the Erlang `lib` directory, these paths become invalid when the node is started against a new release of `sse_core` since the individual app directory names change.

This pull request allows a YAWS docroot to be specified either as a literal href (if relative, resolved against the node directory) or as an `{M, F, A}` triple. In the latter case, the M:F(A) is invoked at runtime. 

For example `{sse_util, priv_dir, [sse_cfg, "/docroot"}` resolves to `/usr/local/lib/sse/lib/sse_cfg_NNNN_XXXX/priv/docroot` where NNNN_XXXX is release-dependent.

lucafavatellawork commented:
> Can the `svc_sequencer`'s docroot be deleted? I see you did not update its config in this PR, it looks not picked up by the configuration at runtime, and its content seems not useful.

Proposed in #146 .

jacobythwaites commented:
> I see the app folder gets the `VERSION` make variable. Ok.
> 
> Can the `svc_sequencer`'s docroot be deleted? I see you did not update its config in this PR, it looks not picked up by the configuration at runtime, and its content seems not useful.

No, leave this in place. We may well need to render the sequencing graph at some point, and this XSL does the job and is exposed at the `/sequencer/graph2dot.xsl` path.

I've deleted the spurious .dot file you found, though, and corrected the svc_sequencer docroot configurator ditto. Many thanks.

lucafavatellawork commented on apps/svc_sequencer/src/svc_sequencer_configurator.erl:
```diff
@@ -57,7 +57,6 @@ configure({sse_yaws, yaws_app_mods}, Current) ->
 % Add svc_sequencer docroot.
 configure({sse_yaws, yaws_doc_roots}, Current) ->
```
I doubt this is configured (see line 37) though we can expose it when needed.
